.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+[Description of PR]
+
+TODO:
+* [ ] Unit tests and/or doctests in docstrings
+* [ ] ``tox -e py36`` passes locally
+* [ ] ``tox -e py27`` passes locally
```

Comment:
Personally I don't think we need to be so formal. Maybe something like this for the first one. Similar for the second.

```markdown
* [ ] Python 3.6 tests pass (e.g. ``tox -e py36``)
```

Reply:
Done